sed/core/processor.py

```diff
@@ -22,7 +26,7 @@
 class SedProcessor:
     """[summary]"""
 
-    def __init__(
+    def __init__(  # pylint: disable=dangerous-default-value
```

Comment:
Instead of disabling the pylint warning, we could correct the initialization to: `metadata = None`, and `if metadata is None: metadata = {}`

Reply:
Good suggestions, thanks.